src/hooks/sounds.tsx

```diff
@@ -35,10 +37,12 @@ export interface ExposedDataWithPlay extends ExposedData {
 
 export interface CustomSoundsState {
   soundConfigs: SoundConfigMap;
+  isCustomSoundsLoaded: boolean;
 }
 
 export const initialValue: CustomSoundsState = {
   soundConfigs: {},
+  isCustomSoundsLoaded: false,
```

Comment:
We can probably just keep this simple, as the context of how we access this variable (eg. from the hooks/etc) will imply that it relates to the custom sounds:

```suggestion
  isLoaded: false,
```